Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/InfectionsCard.kt

```diff
@@ -31,11 +34,67 @@ class InfectionsCard(parent: ViewGroup) :
         }
 
         with(item.stats as InfectionStats) {
+
+            infectionsContainer.contentDescription =
+                buildAccessibilityStringForInfectionsCard(item.stats, newInfections, sevenDayAverage, total)
+
             primaryLabel.text = getPrimaryLabel(context)
             primaryValue.text = formatStatisticalValue(context, newInfections.value, newInfections.decimals)
+            primaryValue.contentDescription = StringBuilder()
+                .append(getPrimaryLabel(context))
+                .append(" ")
+                .append(formatStatisticalValue(context, newInfections.value, newInfections.decimals))
+                .append(" ")
+                .append(context.getString(R.string.statistics_card_infections_title))
+
             secondaryValue.text = formatStatisticalValue(context, sevenDayAverage.value, sevenDayAverage.decimals)
+            secondaryValue.contentDescription = StringBuilder()
+                .append(context.getString(R.string.statistics_card_infections_secondary_label))
+                .append(" ")
```

Comment:
You could use some extension function to make this nicer. E.g. `appendWhiteSpace()` and `appendLineBreak()`. You might even want to take it to the next level
`StringBuilder().concatStringWithWhiteSpace(context, R.string.id1, R.string.id2, R,string.id3, ...)`

Reply:
PS: it is already there..  (appendln)
called String.join or StringJoiner

Reply:
@ralfgehrer added two extension functions `StringBuilder().appendWithWhiteSpace()` and `StringBuilder().appendWithLineBreak()` - chose the term `append` to be aligned with the default `append()` function.

@BMItter good point with the `appendLine()`, normally I would have used this but since I ran into some issues with TalkBack on some devices when only appending `"\n"` I need to append `" \n "` which is not supported by default afaik.